guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -4,7 +4,20 @@
 [id="foreman-headline-features"]
 == Headline Features
 
-There are no highlights with Foreman {ProjectVersion}.
+=== IPv6 Support
+
+Deployment and operation in IPv6-only networks is now fully supported.
+
+Provisioning over IPv6 is supported on bare metal hosts.
+For compute resources, you can define the machine outside Foreman on the compute resource and then you can provision the machine as bare metal in Foreman.
+
+=== Secure Boot provisioning
+
+Support for Secure Boot provisioning on bare-metal, VMware vSphere and Libvirt across multiple operating systems.
```

Comment:
@nofaralfasi is this a good addition? I want to also imply that VMs will be created with Secure Boot enabled.
```suggestion
Provisioning with Secure Boot is now supported on bare-metal, VMware vSphere and Libvirt across multiple operating systems.
New virtual machines will be created with the default keys enrolled.
```

Reply:
I don't quite follow. Are you referring to all new VMs being created with the default keys enrolled (which isn't the case)? Or are you talking specifically about new VMs with Secure Boot enabled? If it's the latter, why is it important to mention that here? Isn't that implicit?

Reply:
LGTM. Thanks!